main.cpp

```diff
@@ -1 +1,239 @@
+#include <iostream>
+#include <string>
+#include <cstdlib>
+#include <ctime>
+#include <vector>
+#include <charconv>
+#include <algorithm>
+#include <iterator>
 
+using namespace std;
+
+void print(int guesses = 0) // print the hangman
+{ 
+    if (guesses >= 1) {cout << "  | ";}
+    if (guesses >= 2) {cout << "\n  | ";}
+    if (guesses >= 3) {cout << "\n  O ";}
+    if (guesses == 4) {cout << "\n /";}
+    if (guesses == 5) {cout << "\n /|";}
+    if (guesses >= 6) {cout << "\n /|\\";}
+    if (guesses == 7) {cout << "\n / ";}
+    if (guesses >= 8) {cout << "\n / \\";}
+    if (guesses >= 9) {cout << "\n ___";}
+    if (guesses == 10) {cout << "\n|   |\nYOU LOST!";}
+}
+
+void printoptions() // print options
+{
+    cout << "\nSelect One The Following Option:"
+         << "\n[1] Guess a letter"
+         << "\n[2] Guess the word"
+         << "\n[3] Check guessed letters"
+         << "\n[4] View options again"
+         << "\n[5] Quit game" << endl
+         << "[6] Check Current Word (FOR TESTING ONLY)\n";
+}
+
+void printword(const string word, const vector<char> cg) // function to print
```

Comment:
I can see you added the 'const', but they are not references. Not sure if you've come across passing by references, take a  quick look at https://www.w3schools.com/cpp/cpp_function_reference.asp